psse/psse-model/src/main/java/com/powsybl/psse/model/io/RecordGroupReaderWriter.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.psse.model.io;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+
+/**
+ * @author Luma Zamarreño <zamarrenolm at aia.es>
+ * @author José Antonio Marqués <marquesja at aia.es>
+ */
+interface RecordGroupReaderWriter<T> {
```

Comment:
I dislike this interface name: I think we should split this interface in two different interfaces.

Reply:
Ok, I am not against splitting readers and writers.